agent/statsd_server.go

```diff
@@ -31,6 +32,38 @@ func NewStatsdServer(a *Agent) *statsd.Server {
 	return s
 }
 
+// NewServer will create a new statsd Server with the default configuration.
+func NewServer() *statsd.Server {
```

Comment:
This change seems unrelated to the PR topic, a mistake?

Reply:
It seems so. NewServer is deprecated in gostatsd version 5.4.1 and we had to update gostatsd for sirupsen/logrus compatibility. I just used the most recent release as that’s what we originally attempted, which prompted this logrus issue.

Reply:
Yes! For some reason I assumed the function was deprecated _before_ the logrus fix. But it wasn't, so we can totally roll this back to 2.4.0.